tests/unit/tools/test_json_utils.py

```
@@ -0,0 +1,144 @@
+    ConcatenatedJSONError,
+    _advance_json_state,
+    _try_collect_object,
+    safe_json_parse,
+    split_concatenated_json,
+    validate_tool_args_safety,
+)
+
+
+class TestAdvanceJsonState:
+    def test_escape_next_true(self):
+        escape, in_str, skip = _advance_json_state("n", True, True)
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)